js/storage.js

```diff
@@ -201,5 +204,6 @@ export function loadSingleUrl(url) {
 
 function postLoadUIUpdate() {
 	editors.setCurrentFile(Object.keys(files)[0]);
-	updateFileList();
+	main.updateFileList();
+	main.compileCode();
```

Comment:
Why does this have to be added?

Reply:
`main` ? or `compileCode` ?

Reply:
`main` becuase updateFileList is in main module (created by moving all code from index.html to main.js)
`main.compileCode` i've added becuase i've noticed that code do not compile if loaded from url

Reply:
The latter: why is this newly called? `main` has to be added due to the new scoping, that much I understand.

Reply:
Okay. I'll split it into a separate commit, then, and make sure not to squash this PR.